stdlib/pydoc.pyi

```diff
@@ -186,7 +186,7 @@ def tempfilepager(text: str, cmd: str) -> None: ...
 def ttypager(text: str) -> None: ...
 def plainpager(text: str) -> None: ...
 def describe(thing: Any) -> str: ...
-def locate(path: str, forceload: bool = ...) -> object: ...
+def locate(path: str, forceload: bool = ...) -> object | None: ...
```

Comment:
`None` is a subtype of `object`

Reply:
That's true. 
Including the `None` would provide better type hinting for users/type checkers, but most of them will not complain, I noticed the SonarLint does on the `is None` check if I use any of these functions. I should probably open an issue on their page.
If we do not accept this we should probably remove these also [stubs/six/six/__init__.pyi#L53](https://github.com/python/typeshed/blob/c661ac3ad6dc32d789e7458654dda49d095b2ba9/stubs/six/six/__init__.pyi#L53) and[stubs/console-menu/consolemenu/console_menu.pyi](https://github.com/python/typeshed/blob/c661ac3ad6dc32d789e7458654dda49d095b2ba9/stubs/console-menu/consolemenu/console_menu.pyi#L23)

Reply:
> Including the `None` would provide better type hinting for users/type checkers

I don't think so -- I think type checkers should just immediately simplify the `object | None` union to `object`, if they're working correctly. Do you have evidence of a situation where a type checker treats `object | None` differently to `object`? I'd be curious to see it.

> If we do not accept this we should probably remove these also [stubs/six/six/**init**.pyi#L53](https://github.com/python/typeshed/blob/c661ac3ad6dc32d789e7458654dda49d095b2ba9/stubs/six/six/__init__.pyi#L53) and[stubs/console-menu/consolemenu/console_menu.pyi](https://github.com/python/typeshed/blob/c661ac3ad6dc32d789e7458654dda49d095b2ba9/stubs/console-menu/consolemenu/console_menu.pyi#L23)

Sure, I'll happily accept a separate PR cleaning those up :)

Reply:
Hmm, yeah, I think SonarLint's behaviour is incorrect there. Might want to file a bug report with them :)